docs/python/python-quick-start.md

```diff
@@ -0,0 +1,125 @@
+---
+Order: 1
+Area: python
+TOCTitle: Quick Start
+ContentId: 77828f36-ae45-4887-b25c-34545edd52d3
+PageTitle: Quick Start Guide for Python in VS Code
+DateApproved: 1/9/2024
+MetaDescription: A quick start guide to get you up and coding with the Python extension in Visual Studio Code.
+MetaSocialImage: images/tutorial/social.png
+---
+
+# Quick Start Guide for Python in VS Code
+
+The Python extension makes VS Code an excellent Python editor, works on any operating system, and is usable with a variety of Python interpreters.
+
+Get started by installing:
+-	[VS Code](https://code.visualstudio.com/)
+-	[A Python Interpreter](https://code.visualstudio.com/docs/python/python-tutorial#_install-a-python-interpreter) (any [actively supported Python version](https://devguide.python.org/#status-of-python-branches))
+-	[Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python) from the VS Code Marketplace
+
+![Gif installing the Python extension in a fresh install of VS Code.](/docs/python/images/quick-start/qs-python-ext-install.gif)
+
+To further customize VS Code for Python, you can leverage the [Python profile template](https://code.visualstudio.com/docs/editor/profiles#_python-profile-template), automatically installing recommended extensions and settings. For Data Scientists, consider using the [Data Science profile template](https://code.visualstudio.com/docs/editor/profiles#_data-science-profile-template).
```

Comment:
```suggestion
To further customize VS Code for Python, you can leverage the [Python profile template](https://code.visualstudio.com/docs/editor/profiles#_python-profile-template), automatically installing recommended extensions and settings. For Data Science projects, consider using the [Data Science profile template](https://code.visualstudio.com/docs/editor/profiles#_data-science-profile-template).
```

Reply:
or "Data related" or something like that. Just to make it more open to those may not identify with being a data scientist